### ISSUE TITLE
Update the version of udi_interface

### ISSUE THREAD

**Issue by bpaauwe** [sjpbailey/udi-enphase-poly-master-v3#1]
This is requiring a fairly old and outdated version of udi_interface.

At this point in the development cycle, there is no guarantee that version 3.0.10 will work properly with the latest version of PG3.  The latest version of udi_interface is 3.0.23.

When new versions of PG3 are released, there is typically a new version of the udi_interface that is required to take advantage of new features.

**Comment by sjpbailey** [comment]
Hello Bob,
 
Oh crap I have not kept up with your upgrades.
There are some times when the poly's do need to be re-started to poll correctly version upgrade!

I need to also update my bascontrolns PyPI Module for Python version and feature upgrade.

I also need to get all of my poly’s as installable instead of public repositories on GitHub, need to figure that out.

 Github crashed last week {?} and I still haven’t tried to clean my commits because that was no longer working after.

Learning and forgetting what has been learned is what I am great at so far, lol!

Highest Regards!

**Comment by sjpbailey** [comment]
Hello Bob,

 I updated UOM for kW and kWh tested and everything came up fine.
Then I remembered to update udi_interface to 3.0.24 from 3.0.10 and now get this exception? ‘Level’ call?

2021-12-04 20:11:56,965 MQTT       udi_interface.interface INFO     interface:_subscribe: MQTT Subscribed Succesfully for Message ID: 1 - QoS: (0,)
2021-12-04 20:11:57,131 MQTT       udi_interface.interface INFO     interface:_message: Successfully set controller :: ST to 1 UOM 2
2021-12-04 20:11:59,731 MainThread udi_interface.interface INFO     interface:addNode: Adding node Enphase Site Controller(controller) [None]
2021-12-04 20:11:59,813 Command    udi_interface      ERROR    udi_interface:write: Exception in thread
2021-12-04 20:11:59,814 Command    udi_interface      ERROR    udi_interface:write: Command
2021-12-04 20:11:59,815 Command    udi_interface      ERROR    udi_interface:write: :
2021-12-04 20:11:59,815 Command    udi_interface      ERROR    udi_interface:write: Traceback (most recent call last):
2021-12-04 20:11:59,816 Command    udi_interface      ERROR    udi_interface:write: File "/usr/local/lib/python3.8/threading.py", line 932, in _bootstrap_inner
2021-12-04 20:11:59,821 Command    udi_interface      ERROR    udi_interface:write: self.run()
2021-12-04 20:11:59,822 Command    udi_interface      ERROR    udi_interface:write: File "/usr/local/lib/python3.8/threading.py", line 870, in run
2021-12-04 20:11:59,825 Command    udi_interface      ERROR    udi_interface:write: self._target(*self._args, **self._kwargs)
2021-12-04 20:11:59,826 Command    udi_interface      ERROR    udi_interface:write: File "/var/polyglot/.local/lib/python3.8/site-packages/udi_interface/interface.py", line 629, in _parseInput
2021-12-04 20:11:59,829 Command    udi_interface      ERROR    udi_interface:write: self._handleInput(key, input[key])
2021-12-04 20:11:59,830 Command    udi_interface      ERROR    udi_interface:write: File "/var/polyglot/.local/lib/python3.8/site-packages/udi_interface/interface.py", line 635, in _handleInput
2021-12-04 20:11:59,835 Command    udi_interface      ERROR    udi_interface:write: self._inConfig(item)
2021-12-04 20:11:59,837 Command    udi_interface      ERROR    udi_interface:write: File "/var/polyglot/.local/lib/python3.8/site-packages/udi_interface/interface.py", line 573, in _inConfig
2021-12-04 20:11:59,841 Command    udi_interface      ERROR    udi_interface:write: if l['level'] not in ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']:
2021-12-04 20:11:59,842 Command    udi_interface      ERROR    udi_interface:write: KeyError
2021-12-04 20:11:59,843 Command    udi_interface      ERROR    udi_interface:write: :
2021-12-04 20:11:59,844 Command    udi_interface      ERROR    udi_interface:write: 'level'
2021-12-04 20:11:59,854 MQTT       udi_interface.interface ERROR    interface:_message: error No valid API calls provided
2021-12-04 20:11:59,855 MQTT       udi_interface.interface ERROR    interface:_message: Invalid command received in message from PG3: 'message' {"getNsInfo":{}}

Any idea where is messed up?

Highest Regards,
Steve
